docs/dev-guide/configJson.md

```diff
@@ -59,3 +59,54 @@ For example, if your IPv4 is `192.168.1.60` and PL is running on port `3000`, yo
   "serverCanonicalHost": "http://192.168.1.60:3000"
 }
 ```
+
+## Workspaces and external graders
+
+You should set the workspace host home directory root and home directory root in your `config.json`.
+
+```json title="config.json"
+{
+  "workspaceHostHomeDirRoot": "/tmp/workspace",
+  "workspaceHomeDirRoot": "/tmp/workspace"
+}
+```
+
+### Running workspaces / external graders natively on Mac OS
+
+If you are running workspaces natively on Mac OS, you may need to change `"workspaceDevContainerHostname"` to "localhost".
+
+```json title="config.json"
+{
+  "workspaceDevContainerHostname": "localhost"
+}
+```
+
+Certain images detect if you are running as root and try to chown the workspace files to the user 1001 before stepping down to the user 1001 (`pl-gosu-helper.sh`). On macOS, this will fail because bind mounts cannot be chown'd (FUSE-based filesystems). To fix this, you can set `"workspaceJobsDirectoryOwnerUid"` and `"workspaceJobsDirectoryOwnerGid"` to 1001 in your `config.json`.
+
+```json title="config.json"
+{
+  "workspaceJobsDirectoryOwnerUid": 1001,
+  "workspaceJobsDirectoryOwnerGid": 1001
+}
+```
+
+### Testing local docker images
+
+When testing [docker images](../dockerImages.md) locally, you may want to force PrairieLearn to use the local version of an image.
+
+```json title="config.json"
+{
+  "workspacePullImagesFromDockerHub": false,
+  "externalGradingPullImagesFromDockerHub": false
+}
+```
+
+## Enterprise
+
+Some features of PrairieLearn are only available in the Enterprise Edition.
```

Comment:
We should make note of and link to the license, which describes who can set this flag and when.

Reply:
I added a note in 8a16f3e.